cli/app.go

```diff
@@ -520,6 +520,155 @@ var app = &cli.App{
 				},
 			},
 		},
+		{
+			Name:      "train",
+			Usage:     "train on data",
+			UsageText: "viam data train [other options]",
+			Subcommands: []*cli.Command{
+				{
+					Name:  "submit",
+					Usage: "submits training job on data in Viam cloud",
+					UsageText: fmt.Sprintf("viam data train submit <%s> [other options]",
```

Comment:
Same here and probably throughout

Reply:
Done! Retested edge cases that hit the error logs (e.g. which suggest using `viam train submit --help`)